docs/apis/commands.md

```diff
@@ -236,3 +239,54 @@ apis:
 ```
 
 Note that `line_end` is exclusive, meaning that you have to add `1` (`0` indexed) to the actual line you want to stop being processed.
+
+### Assert output exists before processing
+
+Add an assert block with a nested variable of `match` and/or `not_match`.
+Note both options use regex.
```

Comment:
Could you add examples that use regex?

Reply:
@Noly the example is regex already, the output is `hi:bye` the match value is `hi` so because it is regex it matches and continues. If it was not regex that assertion and example would not continue as documented.

Reply:
I see, I'm curious on what would happen if customer set up smt like: 
 `match: !hi `

It would continue with all outputs that doesn't match with hi? 🤔

Reply:
`!hi` would not match, and the assertion would fail.

Also the way this regex is defined is implemented the same way as the other existing features in Flex that also use regex. 
So for users the experience will ultimately be consistent and the same for them in other places of Flex as well.

Reply:
👍 Make sense. Thanks for the explanation!